packages/core/test/actorLogic.test.ts

```diff
@@ -232,6 +232,247 @@ describe('promise logic (fromPromise)', () => {
 
     createActor(promiseLogic).start();
   });
+
+  it('should abort when stopping', async () => {
+    const deferred = defer<number>();
+    const fn = jest.fn();
+    const promiseLogic = fromPromise((ctx) => {
+      return deferred.promise.then((res) => {
+        fn(ctx.signal.aborted);
+        return res;
+      });
+    });
+
+    const actor = createActor(promiseLogic).start();
+
+    actor.stop();
+
+    deferred.resolve(42);
+    await deferred.promise;
+    expect(fn).toHaveBeenCalledWith(true);
```

Comment:
I was thinking about something closer to this:
```suggestion
    const aborted = jest.fn();
    const promiseLogic = fromPromise(({ signal }) => {
      return new Promise(() => {
        signal.on("abort", aborted);
      })
    });

    const actor = createActor(promiseLogic).start();

    actor.stop();

    expect(aborted).toHaveBeenCalledWith(true);
```